service.yml

```diff
@@ -8,3 +8,4 @@ github:
   repo_name: confluentinc/confluent-kafka-python
 semaphore:
   enable: true
+  run_semaphore_public: true
```

Comment:
i don't think this is the right place to allow this configuration.  it might make sense in `projects.json`, but since the team owning this repo can approve/merge a PR adding this config, it wouldn't really provide any roadblock.